indico/modules/events/timetable/client/js/EntryPopup.tsx

```diff
@@ -160,15 +189,35 @@ function EntryPopupContent({
             <div>{title}</div>
           </div>
         </Card.Header>
-        {/* TODO: (Ajob) Replace dummy data below */}
         <CardItem icon="clock outline">{formatTimeRange('en', startTime, endTime)}</CardItem>
-        <CardItem icon="map marker alternate">Room 101</CardItem>
-        {type !== EntryType.Break && (
-          <>
-            <CardItem icon="microphone">John Doe</CardItem>
-            <CardItem icon="file powerpoint outline">slides.pptx</CardItem>
-          </>
-        )}
+        {locationArray.length ? (
+          <CardItem icon="map marker alternate">
+            <List style={{textAlign: 'left', marginTop: 0}}>
+              {locationArray.map((v: string, i) => (
+                <List.Item color="red" key={i}>
+                  {v}
+                </List.Item>
+              ))}
+            </List>
+          </CardItem>
+        ) : null}
+        {presenters?.length ? (
+          <CardItem icon="microphone">
+            {presenters.join(', ') || Translate.string('No presenters')}
```

Comment:
We could also use [Intl.ListFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) here

Reply:
Hmmm, I think it's a lot of code just to have 'and' at the last element, especially considering the context. It's also a little less readable imo.

I discussed it briefly with @ThiefMaster and we agreed that if we are gonna start using this, maybe it's better to make a string sum wrapper for it in general so that we don't pass the locale etc every time.